core/silkworm/chain/validity.cpp

```diff
@@ -106,6 +106,29 @@ ValidationResult validate_block_header(const BlockHeader& header, const StateBuf
         }
     }
 
+    // Ethash PoW verification
+    if (header.mix_hash) {
```

Comment:
Please add an option to ChainConfig allowing to opt out of PoW validation instead. We can opt out of PoW validation for tests, but not for `kMainnetConfig`. Otherwise there is a security risk that validate_block_header passes bogus blocks on the mainnet.

Reply:
Right. Will do

Reply:
Done. How to reinclude the tests removed due to #144 ?
(I manually tested them adding the files and they're ok)